src/PowerShell/PowerShellManager.cs

```diff
@@ -72,6 +74,9 @@ internal PowerShellManager(ILogger logger)
             _pwsh.Streams.Verbose.DataAdding += streamHandler.VerboseDataAdding;
             _pwsh.Streams.Warning.DataAdding += streamHandler.WarningDataAdding;
 
+            // Install function app dependent modules
+            initAction?.Invoke(_pwsh, logger);
```

Comment:
change this to async

Reply:
This is an synchronous call, so the calling thread has to wait. Will keep it as is.